dataeng/jobs/analytics/AnalyticsExporter.groovy

```diff
@@ -129,11 +133,12 @@ class AnalyticsExporter {
         }
 
         dslFactory.job('analytics-exporter-master') {
+            description('The Analytics Exporter weekly job, which exports tons of structure and state data for every course for every org and delivers them encrypted to our partners via S3.  Specifically, this sets up the shared edx-platform execution environment, fetches a list of all the orgs, then kicks off downstream analytics-exporter-worker jobs for each one.')
             parameters {
                 stringParam('ORGS', '*', 'Space separated list of organizations to process. Can use wildcards. e.g.: idbx HarvardX')
                 stringParam('EXPORTER_BRANCH', 'environment/production', 'Branch from the edx-analytics-exporter repository. For tags use tags/[tag-name].')
-                stringParam('PLATFORM_BRANCH', 'aed/analytics-exporter-settings-hotfix', 'Branch from the edx-platform repository. For tags use tags/[tag-name].')
-                stringParam('CONFIG_FILENAME', 'default.yaml', 'Name of configuration file in analytics-secure/analytics-exporter.')
+                stringParam('PLATFORM_BRANCH', 'tags/release-2020-08-20-14.08', 'Branch from the edx-platform repository. For tags use tags/[tag-name].')
```

Comment:
This branch tag does match the above which is tags/release-2020-09-08-16.55

Reply:
good catch